### PR TITLE
Added the glossary tooltips to Virtual IPs and Service Proxies

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref: #38166

### CODE REVIEW

File: content/en/docs/reference/networking/virtual-ips.md [hunk 1]
```diff
@@ -134,11 +139,13 @@ attempts to resynchronize iptables rules with the kernel. If it is
...
 small clusters, but it results in a lot of redundant work when lots of
 things change in a small time period. For example, if you have a
-Service backed by a Deployment with 100 pods, and you delete the
+Service backed by a {{< glossary_tooltip term_id="deployment" text="Deployment" >}}
+with 100 pods, and you delete the
 Deployment, then with `minSyncPeriod: 0s`, kube-proxy would end up
 removing the Service's Endpoints from the iptables rules one by one,
 for a total of 100 updates. With a larger `minSyncPeriod`, multiple
-Pod deletion events would get aggregated together, so kube-proxy might
+Pod deletion {{< glossary_tooltip term_id="event" text="events" >}} would get aggregated
```
Comment (sftim) [inline content/en/docs/reference/networking/virtual-ips.md:147]:
```suggestion
Pod deletion events would get aggregated
```
we're not talking about the Event API here.

File: content/en/docs/reference/networking/virtual-ips.md [hunk 2]
```diff
@@ -274,7 +282,7 @@ someone else's choice.  That is an isolation failure.
 In order to allow you to choose a port number for your Services, we must
 ensure that no two Services can collide. Kubernetes does that by allocating each
 Service its own IP address from within the `service-cluster-ip-range`
-CIDR range that is configured for the API server.
+CIDR range that is configured for the {{< glossary_tooltip term_id="kube-apiserver" text="API Server." >}}
```
Comment (sftim) [inline content/en/docs/reference/networking/virtual-ips.md:285]:
```suggestion
CIDR range that is configured for the {{< glossary_tooltip term_id="kube-apiserver" text="API Server" >}}.
```

**Review by sftim — COMMENTED** [review]
Thanks

One important tweak (about events)

File: content/en/docs/reference/networking/virtual-ips.md (hunk 1 above)
Comment (vaibhav2107) [reply content/en/docs/reference/networking/virtual-ips.md:147]: Thanks @sftim for the info, will update the change

**Comment by vaibhav2107** [comment]
PTAL @sftim

**Comment by vaibhav2107** [comment]
Hi @nate-double-u @shannonxtreme 
Could you please have a look

**Comment by shannonxtreme** [comment]
@vaibhav2107 when reviewers get assigned to an issue or PR they get notified through GitHub so you don't have to ping them. Since most are volunteers reviews might take longer than you expect 🙂

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by vaibhav2107** [comment]
Sure @shannonxtreme, will keep in mind

**Comment by reylejano** [comment]
Preview: https://deploy-preview-39307--kubernetes-io-main-staging.netlify.app/docs/reference/networking/virtual-ips/

/approve

**Pull request merged**